pkg/reconciler.v1/common/service.go

```diff
@@ -72,40 +72,40 @@ func (r *KubeflowServiceReconciler) OverrideForServiceInterface(ui ReconcilerUti
 }
 
 // GetPortsFromJob gets the ports of job container. Port could be nil, if distributed communication strategy doesn't need and no other ports that need to be exposed.
-func (r *KubeflowServiceReconciler) GetPortsFromJob(spec *commonv1.ReplicaSpec) (map[string]int32, error) {
+func (r *ServiceReconciler) GetPortsFromJob(spec *commonv1.ReplicaSpec) (map[string]int32, error) {
 	defaultContainerName := r.GetDefaultContainerName()
 	return core.GetPortsFromJob(spec, defaultContainerName)
 }
 
 // GetServicesForJob returns all services associated with this job
-func (r *KubeflowServiceReconciler) GetServicesForJob(ctx context.Context, job client.Object) ([]*corev1.Service, error) {
+func (r *ServiceReconciler) GetServicesForJob(ctx context.Context, job client.Object) ([]*corev1.Service, error) {
 	svcList := &corev1.ServiceList{}
 	err := r.List(ctx, svcList, client.MatchingLabels(r.GenLabels(job.GetName())))
 	if err != nil {
 		return nil, err
 	}
 
 	var svcs []*corev1.Service = nil
```

Comment:
Can we do `var svcs []*corev1.Service`?  it's a little bit weird to explicitly assign nil here.

Reply:
sure. also changed for `var pods []*corev1.Pod`.